rclpy/src/rclpy/_rclpy_logging.c

```diff
@@ -199,6 +199,54 @@ rclpy_logging_severity_level_from_string(PyObject * Py_UNUSED(self), PyObject *
   return PyLong_FromLongLong(severity);
 }
 
+/// Get log unset severity level as int from rcutils
+/// \return RCUTILS_LOG_SEVERITY_UNSET
```

Comment:
@BMarchi nit^2: `\return RCUTILS_LOG_SEVERITY_UNSET as a PyLong`, here and everywhere else.